src/coreclr/pal/src/configure.cmake

```diff
@@ -1032,6 +1032,15 @@ int main(int argc, char **argv)
 check_symbol_exists(unw_get_save_loc libunwind.h HAVE_UNW_GET_SAVE_LOC)
 check_symbol_exists(unw_get_accessors libunwind.h HAVE_UNW_GET_ACCESSORS)
 
+check_cxx_source_compiles("
+#include <libunwind.h>
+
+int main(int argc, char **argv)
```

Comment:
It would be better to not to use compilation check, as it would require adding this `HAVE_UNW_AARCH64_X19` to tryrun.cmake. We should use `check_symbol_exists` instead (I hope it will work for this symbol on both mac and Linux) which cmake can perform against target frame headers even in cross build.

Reply:
I think I tried, but `check_symbol_exists` did not work for some reason. I concluded it does not work for enum elements, but perhaps I just used it incorrectly.

Reply:
I basically just did:
`check_symbol_exists(UNW_AARCH64_X19 libunwind.h HAVE_UNW_AARCH64_X19)`

similar to the other checks above, but maybe this is not the way for enums?

check_cxx_source_compiles works though.

Reply:
Ah, I guess this is the reason:
> CheckSymbolExists
Provides a macro to check if a symbol exists as a function, variable, or macro in C

Actually, I was wrong, the check_cxx_source_compiles is actually still performed dynamically in cross build, the one that is not is the `CheckCXXSourceRuns`. So this is ok.